chromium_src/components/search_engines/brave_template_url_prepopulate_data_unittest.cc

```diff
@@ -63,6 +63,15 @@ class BraveTemplateURLPrepopulateDataTest : public testing::Test {
     TemplateURLPrepopulateData::RegisterProfilePrefs(prefs_.registry());
   }
 
+  void CheckForCountry(char digit1, char digit2, std::string expected) {
```

Comment:
Any reason not to pass the last param as `const std::string& expected`?

Reply:
Fixed with https://github.com/brave/brave-core/pull/3591/commits/201436c15a1660d11fc108ac1495933e2f8358ea 👍